### ISSUE TITLE
New Logo Design for Minimal Todo

### ISSUE THREAD

**Issue by sametceylan** [avjinder/Minimal-Todo#81]
Hello sir, my name is Samet Ceylan and I am a designer. If you want and let me, I want to design a creative logo for Minimal Todo. I want you to know that I will do it for you. I do not want any compensation. If you would like me to design and if you have any particular detail, please contact me.

**Comment by avjinder** [comment]
@sametceylan that would be great. I'm excited to see what you come up with.

**Comment by sametceylan** [comment]
@avjinder well. Do you have an idea or task about logo? I can create as your requests

**Comment by avjinder** [comment]
@sametceylan I'd like to see what you come up with. Surprise us all.

**Comment by BUInvent** [comment]
@sametceylan Hey could you possibly think of making a logo/icon for my app Jurassic Gains? That would be cool. Thanks!

**Comment by munadikiehl** [comment]
Hi @BUInvent. Hi, my name is Munadi Kiehl. I'm a graphic designer and I would like to collaborate on your open source project and to propose free icon/logo design. If you're interested in, please, let me know.

Best Regard,
Munadi Kiehl

**Comment by BUInvent** [comment]
Yes @munadikiehl. Yes I most definitely am interested. Thanks for asking!

**Comment by BUInvent** [comment]
@munadikiehl Maybe we should setup a chat somewhere like on Google Hangouts. Let me know.

**Comment by munadikiehl** [comment]
Hi @BUInvent. Thanks for your feedback. Do you have discord to continue our discussion.

**Comment by BUInvent** [comment]
Yes I do we will have to exchange discord #'s. Mine is #3398

**Comment by munadikiehl** [comment]
Hmm... didn’t work. I cannot find your account. May be you can try invite my account : munadikiehl#0187

**Comment by BUInvent** [comment]
Just sent friend request
